src/schema/artwork/__tests__/artwork.test.js

```diff
@@ -1195,8 +1195,32 @@ describe("Artwork type", () => {
       })
     })
 
-    it("is set to free domestic shipping when domestic_shipping_fee_cents is null and international_shipping_fee_cents is present", () => {
+    it("is set to international shipping only when its international_shipping_fee_cents is present and domestic_shipping_fee_cents is null", () => {
       artwork.domestic_shipping_fee_cents = null
+      artwork.international_shipping_fee_cents = 1000
+      return runQuery(query, rootValue).then(data => {
+        expect(data).toEqual({
+          artwork: {
+            shippingInfo: "Shipping: $10 rest of world only",
```

Comment:
This is awkward copy, and we don't enforce international shipping only in reaction - should this be ticketed, or treated as an edge case for now?